addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_animation_sampler_keyframes.py

```diff
@@ -197,10 +201,10 @@ def all_equal(lst):
                                      "Baking animation because of differently located keyframes in one channel")
         return True
 
-    # Baking is required when the animation targets a quaternion with bezier interpolation
-    if channels[0].data_path == "rotation_quaternion" and interpolation == "BEZIER":
-        gltf2_io_debug.print_console("WARNING",
-                                     "Baking animation because targeting a quaternion with bezier interpolation")
-        return True
+    # # Baking is required when the animation targets a quaternion with bezier interpolation
+    # if channels[0].data_path == "rotation_quaternion" and interpolation == "BEZIER":
```

Comment:
Is this comment and code still needed?  Future readers will be confused by the comment saying something is required, followed by code to do it being commented out.

If it's really not needed anymore, let's remove it.

Reply:
forgot about that, thanks!
I'll add the removal to the "prune unused code" pull request

Reply:
Thanks!  #397